.github/workflows/release-desktop-snap.yml

```
@@ -22,7 +22,10 @@
         uses: actions/checkout@v3
         with:
           lfs: true
-      
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libudev-dev 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)